05-classification.Rmd

```diff
@@ -511,7 +533,7 @@ library(tidymodels)
 
 Let's again suppose we have a new observation with perimeter 0 and concavity
```

Comment:
I think we need a sentence or two at the beginning of this paragraph that says something like "Let's now walk through how to use the `tidymodels` packages to perform K-nearest neighbours classification. We will use the cancer data set from above, using the perimeter and concavity variables and $K = 5$ to build our classifier. Then we can use it to predict the diagnosis label for a new observation with perimeter 0 and concavity 3.5, where the diagnosis is unknown..."

Reply:
done